pks/init/setenv.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+
+# ======================================= FUNCTIONS START =======================================
+
+function pks_authenticate_and_target() {
+  [ -z "$PKS_CLUSTER_NAME" ] && { echo "Environment variable PKS_CLUSTER_NAME must be set"; exit 1; }
+  [ -z "$PKS_ENDPOINT" ] && { echo "Environment variable PKS_ENDPOINT must be set"; exit 1; }
+  [ -z "$PKS_USERNAME" ] && { echo "Environment variable PKS_USERNAME must be set"; exit 1; }
+  [ -z "$PKS_PASSWORD" ] && { echo "Environment variable PKS_PASSWORD must be set"; exit 1; }
+
+  echo "Using PKS cluster: $PKS_CLUSTER_NAME"
+
+  pks login -a $PKS_ENDPOINT -u $PKS_USERNAME -p $PKS_PASSWORD --skip-ssl-validation
```

Comment:
What if someone runs the script against a cluster with valid certs? Do we need to add a note about `--skip-ssl-validation`?

Reply:
id say we can just add that flag when we have a cluster with certs